examples/templates-minijinja/src/main.rs

```diff
@@ -0,0 +1,89 @@
+//! Run with
+//!
+//! ```not_rust
+//! cargo run -p example-templates-minijinja
+//! ```
+//! Demo for the MiniJinja templating engine.
+//! Exposes three pages all sharing the same layout with a minimal nav menu.
+
+use axum::extract::State;
+use axum::http::StatusCode;
+use axum::{response::Html, routing::get, Router};
+use minijinja::{context, Environment};
+use std::sync::Arc;
+
+struct AppState<'a> {
+    env: Environment<'a>,
```

Comment:
```suggestion
struct AppState {
    env: Environment<'static>,
```

Since non-static in state is pretty uncommon (might even be completely unsupported, can't remember right now).